README.md

```diff
@@ -39,7 +39,7 @@ Works with mobile versions too.
 A template to build an Android client for your LibreSpeed installation is available [here](https://github.com/librespeed/speedtest-android).
 
 ## Docker
-A docker image is available on the [Docker Hub](https://registry.hub.docker.com/r/adolfintel/speedtest), see `doc_docker.md` for more info about it
+A docker image is available on the [Docker Hub](https://registry.hub.docker.com/r/adolfintel/speedtest), see [doc_docker](doc_docker.md) for more info about it
```

Comment:
Would it be worth making the link text `doc_docker.md`. This would mean that the user knows what file they are going to when looking in the project structure.
```suggestion
A docker image is available on the [Docker Hub](https://registry.hub.docker.com/r/adolfintel/speedtest), see [doc_docker.md](doc_docker.md) for more info about it
```